package.json

```diff
@@ -16,6 +16,8 @@
     "test": "npm run lint && jest",
     "jest": "jest",
     "build": "npm run build-tsc && npm run build-extra && npm run build-webpack",
+    "clean": "node ./scripts/clean.js",
+    "rebuild": "npm run clean && npm run build-tsc && npm run build-extra && npm run build-webpack",
```

Comment:
Please change this to ``npm run clean && npm run build``.

Reply:
Done